src/goDeclaration.ts

```diff
@@ -39,8 +39,10 @@ export function definitionLocation(document: vscode.TextDocument, position: vsco
 		// Assume it's > Go 1.5
 		if (toolForDocs === 'godoc' || (ver && (ver.major < 1 || (ver.major === 1 && ver.minor < 6)))) {
 			return definitionLocation_godef(document, position, offset, includeDocs);
+		} else if (toolForDocs == 'guru') {
```

Comment:
There are linting errors that is breaking the build. 
- `==` should be `===` in line 42
- There is a trailing space in line 45